pkg/admissioncontroller/webhooks/admission/seedrestriction/admission.go

```diff
@@ -144,6 +149,29 @@ func (h *handler) admitBackupEntry(ctx context.Context, seedName string, request
 	return h.admit(seedName, backupBucket.Spec.SeedName)
 }
 
+func (h *handler) admitCertificateSigningRequest(seedName string, request admission.Request) admission.Response {
+	if request.Operation != admissionv1.Create {
+		return admission.Errored(http.StatusBadRequest, fmt.Errorf("unexpected operation: %q", request.Operation))
+	}
+
+	csr := &certificatesv1beta1.CertificateSigningRequest{}
+	if err := h.decoder.Decode(request, csr); err != nil {
+		return admission.Errored(http.StatusBadRequest, err)
+	}
+
+	x509cr, err := utils.DecodeCertificateRequest(csr.Spec.Request)
+	if err != nil {
+		return admission.Errored(http.StatusBadRequest, err)
+	}
+
+	if !gutil.IsSeedClientCert(x509cr, csr.Spec.Usages) {
+		return admission.Errored(http.StatusForbidden, fmt.Errorf("can only create CSRs for seed clusters"))
```

Comment:
For my understanding: doesn't that mean that this admission webhook will deny ALL CSR CREATE requests that are not created for Gardenlets?
Does that block usage of CSRs globally e.g manually to get an x509 for another user & group from the API server?
--> is that desired?

Reply:
> doesn't that mean that this admission webhook will deny ALL CSR CREATE requests that are not created for Gardenlets?
Does that block usage of CSRs globally e.g manually to get an x509 for another user & group from the API server?

No, the `SeedRestriction` handler only handles requests from gardenlets and ignores everything else.

Reply:
ok, gotcha. Found the code responsible. Thanks